Loop/Managers/Alerts/AlertStore.swift

```diff
@@ -128,6 +128,66 @@ extension AlertStore {
 
 extension AlertStore {
 
+    public struct QueryAnchor: RawRepresentable, Equatable {
+        public typealias RawValue = [String: Any]
+        internal var modificationCounter: Int64
+        public init() {
+            self.modificationCounter = 0
+        }
+        public init?(rawValue: RawValue) {
+            guard let modificationCounter = rawValue["modificationCounter"] as? Int64 else {
+                return nil
+            }
+            self.modificationCounter = modificationCounter
+        }
+        public var rawValue: RawValue {
+            var rawValue: RawValue = [:]
+            rawValue["modificationCounter"] = modificationCounter
+            return rawValue
+        }
+    }
+    typealias QueryResult = Result<(QueryAnchor, [StoredAlert]), Error>
+
+    func executeAlertQuery(fromQueryAnchor queryAnchor: QueryAnchor? = nil, limit: Int = 100, completion: @escaping (QueryResult) -> Void) {
```

Comment:
[nit] I didn't include defaults on the other store versions of execute*Query because I wanted any caller to explicitly set the `queryAnchor` (with full understanding what `nil` means and that it is only used in one particular case, that of the very first query) and `limit` (without indicating that any default size was preferred as it entirely depends upon the caller's intention).

Reply:
IMHO, I think having a `nil` default for `queryAnchor` makes sense, as "start from the top by default" seems legit to me.
I do agree, however, that having a default `limit` of 100 is arbitrary and might be surprising.  I'll get rid of that default.